modular_skyrat/modules/customization/modules/surgery/bodyparts/parts.dm

```diff
@@ -0,0 +1,20 @@
+/// Self Destructing Bodyparts, For Augmentation. I'm leaving out heads + chests as, while it would be cool for synths, I also don't want people to start the round unrevivable sans botany because they're dumb as heck. You know who and what you are.
+/obj/item/bodypart/arm/left/selfdestruct/try_attach_limb(mob/living/carbon/limb_owner, special)
+	. = ..()
+	drop_limb()
+	qdel(src)
+
+/obj/item/bodypart/arm/right/selfdestruct/try_attach_limb(mob/living/carbon/limb_owner, special)
+	. = ..()
+	drop_limb()
+	qdel(src)
+
+/obj/item/bodypart/leg/left/selfdestruct/try_attach_limb(mob/living/carbon/limb_owner, special)
+	. = ..()
+	drop_limb()
+	qdel(src)
+
+/obj/item/bodypart/leg/right/selfdestruct/try_attach_limb(mob/living/carbon/limb_owner, special)
+	. = ..()
+	drop_limb()
+	qdel(src)
```

Comment:
> My only concern is, what happens in the prefs menu? We try to avoid deleting stuff when in the prefs menu, and only doing visual changes as needed, without actually deleting the underlying things. I'm just worried that this doesn't follow what the rest of it might be following. Otherwise, sounds good to me.

Renders fine in the preferences menu.

https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/ed692c4e-ead6-4eaa-873b-f3999407fcb0

Reply:
How does it work with markings? Does it save them?
[I need this feature so bad]

Reply:
It remains in your prefs (and visible) but you don't spawn with the marking as your limb is gone.